bin/install_cmdstan

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+"""
+Download and install a CmdStan release from GitHub.
+Optional command line arguments:
+   -v, --version : version, defaults to latest
+   -d, --dir : install directory, defaults to '~/.cmdstanpy
+"""
+import argparse
+import contextlib
+import os
+import os.path
+import re
+import shutil
+import subprocess
+import sys
+import tarfile
+import urllib.request
+
+
+@contextlib.contextmanager
+def pushd(new_dir):
+    previous_dir = os.getcwd()
+    os.chdir(new_dir)
+    yield
+    os.chdir(previous_dir)
+
+
+def usage():
+    print(
+        """Arguments:
+        -v (--version) :CmdStan version
+        -d (--dir) : install directory
+        -h (--help) : this message
+        """)
+
+
+def install_version(cmdstan_version):
+    with pushd(cmdstan_version):
+        print('Building {} binaries'.format(cmdstan_version))
+        cmd = ['make', 'build']
+        proc = subprocess.Popen(
+            cmd, cwd=None, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+        )
+        proc.wait()
+        stdout, stderr = proc.communicate()
+        if proc.returncode:
+            print('Command "make build" failed')
+            if stderr:
+                print(stderr.decode('ascii').strip())
```

Comment:
`utf-8`?

Reply:
good catch!  changed to utf-8 everywhere.